harness/determined/common/experimental/trial.py

```diff
@@ -183,9 +183,8 @@ def list_checkpoints(
         self,
         sort_by: Optional[Union[str, checkpoint.CheckpointSortBy]] = None,
         order_by: Optional[checkpoint.CheckpointOrderBy] = None,
-        limit: Optional[int] = None,
     ) -> List[checkpoint.Checkpoint]:
-        """Returns an iterator of sorted :class:`~determined.experimental.Checkpoint` instances.
+        """Returns a list of sorted :class:`~determined.experimental.Checkpoint` instances.
```

Comment:
`model-dev-guide/model-management/checkpoints.html#querying-checkpoints` has some outstanding references to `limit`.